test/farm/helpers.bash

```diff
@@ -0,0 +1,27 @@
+# -*- bash -*-
+
+load ../system/helpers.bash
+
+function setup(){
+    basic_setup
+}
+
+function teardown(){
+    basic_teardown
+}
+
+# getArch converts the platform hardware arch to the value
+# we expect to see in the manifest list
+function getArch() {
```

Comment:
Would `run_podman info --format '{{.Host.Arch}}' ` work here instead?

Not worth a repush only for this.

Reply:
Fixed